Backpack/Icon/Classes/BPKIconView.m

```diff
@@ -37,7 +37,7 @@ - (instancetype)initWithIconName:(nullable BPKIconName)iconName size:(BPKIconSiz
 
     if (self) {
         self.contentMode = UIViewContentModeScaleAspectFit;
-        self.size = size;
+        _size = size;
```

Comment:
Setting size using `_` so that it does not invoke the subclass `setSize` methods.